web/src/main/java/co/mobileaction/example/web/controller/AdminController.java

```diff
@@ -29,4 +33,12 @@ public ResponseEntity<Boolean> createQueueRequests()
 
         return ResponseEntity.ok(true);
     }
+
+
+    @PostMapping("queue/users")
+    public ResponseEntity<Boolean> createUserQueueRequests() {
+        userQueueService.sendUserRequests(userService.getDistinctUserIds());

```

Comment:
As far as I remember, we need to get distinct userIds from post table (post repository)